### ISSUE TITLE
Can't sync with RC5 (DUPLICATE_PEER_ID)

### ISSUE THREAD

**Issue by Messier81z** [semuxproject/semux#373]
Can't sync with RC5.

#### System information

Semux version: semux-linux-1.0.0-rc.5

OS & Version: Ubuntu 16.04

#### Expected behaviour

Expected to have peers > 0 and syncing chain.

#### Actual behaviour

Peers at 0, not syncing.
Console shows DUPLICATE_PEER_ID error.

#### Steps to reproduce the behaviour

0. No semux wallets running.
1. Download semux-linux-1.0.0-rc.5
2. run it
3. it's stuck with 0 peers.

If I repeat the same with RC3 it works but it does not work in RC5.

#### Backtrace
```
09:38:15.081 [INFO]   PeerClient       Use IP address: 192.168.1.3
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil (file:semux-linux-1.0.0-rc.5/semux.jar) to constructor java.nio.DirectByteBuffer(long,int)
WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
09:38:15.123 [INFO]   PeerClient       Starting IP refresh thread
09:38:15.233 [INFO]   NodeManager      Node manager started
09:38:15.251 [INFO]   PeerServer       Starting peer server: address = 0.0.0.0:5161
09:38:15.294 [INFO]   SemuxAPI         Starting API server: address = 127.0.0.1:5171
09:38:15.303 [INFO]   SemuxBFT         Consensus started
09:38:15.305 [INFO]   SemuxBFT         Entered new_height: height = 1, # validators = 4
09:38:15.341 [DEBUG]  SemuxChannelInitializer New outbound channel: remoteAddress = 127.0.0.1:5161
09:38:15.351 [DEBUG]  ChannelManager   Channel added: remoteAddress = 127.0.0.1:5161
09:38:15.355 [DEBUG]  SemuxP2pHandler  P2P handler active, cid = 0
09:38:15.356 [DEBUG]  SemuxChannelInitializer New inbound channel: remoteAddress = 127.0.0.1:45888
09:38:15.356 [DEBUG]  ChannelManager   Channel added: remoteAddress = 127.0.0.1:45888
09:38:15.357 [DEBUG]  SemuxP2pHandler  P2P handler active, cid = 1
09:38:15.381 [DEBUG]  MessageQueue     Disconnect: reason = DUPLICATE_PEER_ID
09:38:15.382 [INFO]   SemuxP2pHandler  Received DISCONNECT msg, reason = DUPLICATE_PEER_ID
09:38:15.382 [DEBUG]  ChannelManager   Channel removed: remoteAddress = 127.0.0.1:45888
09:38:15.382 [DEBUG]  ChannelManager   Channel removed: remoteAddress = 127.0.0.1:5161
```

**Comment by semuxproject** [comment]
You need to close the old wallet before opening the new one.

The old wallet maintains a connection to remote nodes, which prevents new connections from being established. This is why you get the DUPLICATE_PEER_ID error.

**Comment by Messier81z** [comment]
I did close the old wallet down. I am 100% sure this was the only instance of the wallet running on my host, and in my network.
If I switch back to RC3 it works. When I close RC3 wallet down and open RC5 instead it is broken with DUPLICATE_PEER_ID error.

**Comment by semuxproject** [comment]
Thanks! are you using different coinbase (shown on the main window)?

**Comment by Messier81z** [comment]
I have tried to select a different coinbase account to see if it makes any difference but unfortunately it has not made any. Still stuck.

**Comment by Messier81z** [comment]
Please resolve this issue. I've re-downloaded RC5 and the issue is fixed.

**Comment by phash** [comment]
I had the same issue yesterday. I had to restart some times, then it worked

**Comment by semuxproject** [comment]
The duplicated_peer_id is caused by config parsing and is fixed by 74ea368adfcb4b380726c7f74e13002da0aa2127

This won't affect the syncing process.
